slides/ds101-worskhop.mdx

```diff
@@ -71,6 +73,24 @@ UI developer with experience in both design and development who specializes in i
 
 ---
 
+<Grid columns={2} gap={6}>
+
+## What this workshops is
+
+- Design systems overview
+- How to start a design system
+- What to include in your design system
+- Organizing design system team and processes
+
+## What this workshop is not
```

Comment:
You suggestion is needed, especially about the "not" part.

Reply:
@okonet

Reply:
This is LGTM